bodhi/tests/server/scripts/test_approve_testing.py

```diff
@@ -154,15 +155,63 @@ def test_exception_handler(self, stdout, exit, remove, comment):
             self.db,
             ('This update can be pushed to stable now if the maintainer wishes'),
             author='bodhi')
-        self.assertEqual(stdout.getvalue(),
-                         f'{update.alias} now meets testing requirements\nThe DB died lol\n')
+
+        self.assertEqual(
+            [call.info(f'{update.alias} now meets testing requirements')],
+            log.mock_calls)
+        self.assertEqual(stdout.getvalue(), 'The DB died lol\n')
+        # Check the logging level when debug is False
+        self.assertEqual(log.getEffectiveLevel(), 20)
```

Comment:
This fails in local tests, it returns 30, i.e. `log.WARNING`. Maybe some other test messes around with logging levels?

I'm not sure what the best way to fix this would be:

* Either intercept/mock out the methods of `models.log` that can set the log level so it isn't actually globally changed in tests.
* Or, instead of using `mock.patch` to get hold of the log object, use the `caplog` fixture of pytest. See  https://docs.pytest.org/en/latest/logging.html#caplog-fixture
* Something else?

Reply:
I have the same problem locally so I suspect something is messing up w/ logs somewhere :(

Reply:
Shall I give #2 (pytest caplog fixture) a try?

Reply:
Feel free to give it a try, but from my quick testing I couldn't make it work, if you've played with it before you may have better luck though :)